### DISCUSSION TITLE
Not able to access glue tables via amundsen

### DISCUSSION THREAD

**Discussion by vivek2319** [amundsen-io/amundsen#1642] [Q&A]
Hello,
  I have deployed amundsen on aws ec2 instance. I am successfully able to launch it and search aws glue tables from it. However, when I click on it I am not able to access / see table data. 

Can anybody please help?

P.S. I am using sample glue loader and glue extractor.py files

**Comment by notadeveloper9** [comment]
I am also facing the same issue.
